ts/packages/anchor/src/coder/borsh/event.ts

```diff
@@ -70,4 +83,26 @@ export class BorshEventCoder implements EventCoder {
     const data = layout.decode(logArr.slice(8));
     return { data, name: eventName };
   }
+
+  get idlAddress(): PublicKey {
+    return new PublicKey(this.address);
+  }
+
+  public static isCPIEventData(buffer: Buffer): boolean {
```

Comment:
`isCPIEventData` and `parseCpiEvents` don't have consistent naming. It would be better to use `isCpiEventData` since we also use "Idl" instead of "IDL" in various places.

```suggestion
  public static isCpiEventData(buffer: Buffer): boolean {
```